src/blocks/components.rs

```diff
@@ -769,167 +769,6 @@ pub fn BlocksSection() -> impl IntoView {
     }
 }
 
-const ESTIMATED_NON_TABLE_SPACE_IN_SUMMARY: usize = 290;
-
-#[component]
-pub fn SummaryPageBlocksSection() -> impl IntoView {
```

Comment:
`SummaryPageBlocksSection` was almost a dupe of `BlocksSection`. I'm not sure which copy was the most up-to-date.

Reply:
The test failures should help you resolve this.